script.js

```diff
@@ -2,6 +2,27 @@ const menu = document.querySelector('.mobile-menu');
 const ham = document.querySelector('.ham');
 const xIcon = document.querySelector('.xIcon');
 const menuIcon = document.querySelector('.menuIcon');
+const main = document.querySelector('.main');
+// const logo = document.querySelector('.bold');
+const header = document.getElementById('logo');
+const projs = document.querySelectorAll('.work');
+
+function projOn() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function_ syntax simpler and cleaner.